packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts

```diff
@@ -0,0 +1,90 @@
+import { AgChart, AgChartOptions, AgChartTheme } from "ag-charts-community"
+import { getData } from "./data";
+
+var myTheme: AgChartTheme = {
+  baseTheme: "ag-default-dark",
+  palette: {
+    fills: ["#5C2983", "#0076C5", "#21B372", "#FDDE02", "#F76700", "#D30018"],
+    strokes: ["gray"],
+  },
+  overrides: {
+    common: {
+      title: {
+        fontSize: 24,
+      },
+      padding: {
+        left: 70,
+        right: 70,
+      },
+      axes: {
```

Comment:
does the polar chart need this theme config? it doesn't have axes

Reply:
The intent is to show the same theme config can be used in both cases.

Reply:
Collapsed the two examples back to one.